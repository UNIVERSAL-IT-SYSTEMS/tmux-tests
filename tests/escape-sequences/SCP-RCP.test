#!/bin/sh

S=$(mktemp)
trap 'rm -f $S' 0 1 15

cat <<EOF >$S
printf "\033[ua"
printf "\033[10;10H\033[1;31mb\033[s"
printf "\033[15;15H\033[0mc"
printf "\033[ud"
printf "\033[20;20H\033[0me"
printf "\033[u\033[Cf"

$TMUX_L wait -S tests
$TMUX_L wait never
EOF

$TMUX_L new -d -x 80 -y 25 -- sh $S
$TMUX_L wait tests\; capturep -ep -S- -E-
