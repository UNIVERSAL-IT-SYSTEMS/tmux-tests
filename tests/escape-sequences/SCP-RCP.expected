a








         [1;31mbdf




[0;39m              c




                   e




